tests/e2e/tools/lib/media-library.js

```diff
@@ -0,0 +1,59 @@
+import {expect} from './test-utils.js';
+
+/**
+ * Searches for images in the Library based on a search term or image id.
+ *
+ * @param {Object} page       - The page object for interacting with the browser.
+ * @param {string} searchTerm - The term to be searched, can be an image id.
+ */
+async function searchImages(page, searchTerm) {
```

Comment:
What if no images are found?
Maybe this could be a problem on line 31 when trying to delete an image.